eclass/tree-sitter-grammar.eclass

```diff
@@ -42,7 +42,9 @@ EXPORT_FUNCTIONS src_compile src_install
 # This internal function determines the ABI version of a grammar library based
 # on the package version.
 _get_tsg_abi_ver() {
-	if ver_test -gt 0.21; then
+	if [[ -n ${TS_ABI} ]]; then
```

Comment:
Please [document](https://devmanual.gentoo.org/eclass-writing/index.html#documenting-eclasses) this variable using eclassdoc tags.